notebooks/remote_functions/remote_function_usecases.ipynb

```diff
@@ -0,0 +1,1134 @@
+{
```

Comment:
<p>Any restrictions to be aware of here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/808/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Nothing that I can think of. I made this use case a bit more generic by chaining function dependencies outside the function body, PTAL.</p>